### PR TITLE
feat(584): hide app-header tooltips when menu is collapsed

### PR DESCRIPTION
**Pull request by petey**
Context:
========
When the header menu is collapsed, tooltips display on the menu center screen which can be confusing.

Objective:
----------
Hide tooltips when the menu is collapsed

Misc:
------
Fixes last verifiable issue in https://github.com/screwdriver-cd/screwdriver/issues/584

### CODE REVIEW

File: app/components/app-header/template.hbs [hunk 1]
```diff
@@ -19,52 +19,56 @@
...
           <span class='icontitle'>Slack Channel</span>
-          {{#bs-tooltip placement="bottom"}}Screwdriver Slack Channel{{/bs-tooltip}}
         </a>
       {{/nav.item}}
       {{#nav.item}}
         <a href="https://github.com/screwdriver-cd" class="icon github">
           {{inline-svg "github" class="img"}}
           <span class='icontitle'>Github</span>
-          {{#bs-tooltip placement="bottom"}}Screwdriver Github Repo{{/bs-tooltip}}
+
```
Comment (ian-fox) [inline app/components/app-header/template.hbs]: Blank line here, other than that looks good.

**Comment by r3rastogi** [comment]
Please add in before/after screenshots to the description

File: app/components/app-header/styles.scss [hunk 2]
```diff
@@ -100,14 +100,17 @@
...
 
 @media screen and (min-width: 962px) {
   .navbar-default .navbar-nav > li > a.icon .icontitle {
     display: none;
   }
 
-  .navbar-form input {
-    width: 326px;
+  .tooltips {
+    display: block;
```
Comment (tkyi) [inline app/components/app-header/styles.scss]: Did you mean `display: none;` here?

Comment (tkyi) [inline app/components/app-header/styles.scss]: Nvm, I ran it locally and it looks fine.

**Comment by tkyi** [comment]
Should we add a tool tip to the search button/bar?

**Comment by r3rastogi** [comment]
@tkyi the search bar already says "Search Screwdriver pipelines". I think a tooltip would be redundant

**Pull request merged**